src/coreclr/tools/Common/Compiler/InstructionSetSupport.cs

```diff
@@ -375,6 +375,10 @@ public bool ComputeInstructionSetFlags(int maxVectorTBitWidth,
 
                 if (_supportedInstructionSets.Contains("avx10v1"))
                     _supportedInstructionSets.Add("avx10v1_v512");
+
+                // Having AVX10V1-V512 enabled, means having AVX10V2-V512 enabled as well.
```

Comment:
Is this backwards? It seems that if avx10v2_v512 is supported then avx10v1_v512 is also supported, but not vice-versa.

Reply:
This one does look incorrect. I believe it should be `if (_supportedInstructionSets.Contains("avx10v2"))` so it matches the AVX10v1 handling just above.

Which is to say that if `Avx10v1` is supported and `AVX512` is supported, then `Avx10v1_V512` is supported.

Reply:
I had deliberately kept this as is. By implication

implication        ,X86   ,AVX10v2              ,AVX10v1
implication        ,X86   ,AVX10v2_V512         ,AVX10v1_V512

And what bruce is saying is correct. If we have ```AVX10v2_V512```, then we would have ```AVX10v1_V512```.  Please correct me if I am understanding it wrong. From the design, we cannot have ```AVX10v1``` and have ```AVX10v2```.

Reply:
Pushed this change.

Reply:
Makes sense. thanks for pointing this out @BruceForstall @tannergooding